src/components/SelectedAnnotation.jsx

```diff
@@ -179,24 +244,31 @@ class SelectedAnnotation extends React.Component {
             <hr />
             <span className="secondary-label">Current Script Type</span>
             <div>
-              <div>
-                <span>&#9664;</span>
-                <span className="text-link">Current Keyboard</span>
-                <span>&#9658;</span>
+              <div className="selected-annotation__script-select">
+                <button onClick={this.previousScript}>&#9664;</button>
+                <button className="text-link" onClick={this.toggleScriptOptions}>{this.props.activeScript.name}</button>
```

Comment:
[minor][note] This doesn't check for the existence of `this.props.activeScript`, but a quick check of the code logic indicates that it shouldn't be possible for `this.props.activeScript` to be null or undefined.